src/Converter/Feature2Mesh.js

```diff
@@ -650,6 +669,8 @@ export default {
                 options.layer = this;
             }
 
+            options.collection = collection;
```

Comment:
Why not use ```context.setCollection(options.collection);``` here instead of each sub functions ?

Reply:
Yes, thanks, good catch
I push a proposal